api/src/badge_logic/get_badge_info.js

```diff
@@ -7,14 +7,18 @@ function mapBadgeToTask (badge, x, cond) {
     const map = {
       roadKms: `Add ${x} more km of roads`,
       roadKmMods: `Modify ${x} more km of roads`,
+      coastlineKms: `Add ${x} more km of coastlines`,
+      coastlineKmMods: `Modify ${x} more km of coastlines`,
+      coastlines_add: `Add ${x} more coastlines`,
+      coastlines_mod: `Modify ${x} more coastlines`,
       buildings: `Build ${x} more buildings`,
       daysInRow: `Map ${x} more consecutive days`,
       josm: `Use JOSM to map an area ${x} more times`,
       hashtags: `Participate in ${x} more campaigns`,
       pois: `Add ${x} more nodes`,
       waterways: `Add ${x} more km of waterways`,
       countries: `Map in ${x} more different countries`,
-      daysTotal: `Map ${x} more days in total`
+      daysTotal: `Map ${x} more days in total`,
```

Comment:
@kamicut I think this extraneous comma was introduced when we resolved the conflicts

Reply:
```suggestion
      daysTotal: `Map ${x} more days in total`
```